bears/js/JSONFormatBear.py

```diff
@@ -56,6 +63,22 @@ def run(self, filename, file,
                 column=int(err_content.group(3)))
             return
 
+        for line_number, line in enumerate(file):
+            line = line.expandtabs(indent_size)
+            if len(line) > max_line_length + 1:
+                yield Result.from_values(
+                    origin=self,
+                    message='Line is longer than allowed.' +
```

Comment:
trailing `+` not needed or wanted.  Python does implicit string concatenation.

Reply:
Okay :+1: